topi/python/topi/nn/shortcut.py

```diff
@@ -0,0 +1,47 @@
+"""Shortcut operators (short-cut connections)."""
+from __future__ import absolute_import as _abs
+import tvm
+import topi
```

Comment:
since it is in topi, should not import topi, but instead consider use relative import

Reply:
after moving files to the vision folder, the ci is failing if i use import with relative paths. locally im able to run the testcases successfully.